deptrack-devops-recipes/src/Devops/Bootstrap/Parasite.hs

```diff
@@ -184,3 +184,34 @@ remotedWith mkAction usr clo host =
                                 , "-l", unpack (userName u), unpack ip
                                 , remoteExecution rpath b64 ]
         remoteExecution rpath b64 = unwords [ "chmod", "+x", rpath, ";", rpath, magicRemoteArgv, unpack b64 , ";"]
+
+
+-- | Remotely callback some `Continued a` on some parasited host.
+--
+remoteContinued :: (Typeable a)
+               => DevOp User -> Continued a -> DevOp ParasitedHost -> DevOp (Remoted a)
+remoteContinued usr cont host =
+  devop fst mkOp (do
```

Comment:
indent 4 chars, then 8 chars for the do block,
alternative: write `devop fsg mkOp $ do` on line 193 after the equal and use no parentheses (that's what I do mostly in other files.